.github/workflows/build.yml

```diff
@@ -32,3 +34,13 @@ jobs:
       - name: Lint Shell Scripts
         if: always()
         run: make lint/shellcheck
+
+      - name: Lint Kubernetes Templates
+        if: always()
+        run: make lint/kubernetes
+        # TODO(jawnsy): fix linter warnings and remove this
+        continue-on-error: true
+
+      - name: Package Helm Chart
```

Comment:
@deansheather Updated the .helmignore and added a build step to package up the chart and show the list of files